server/commands/VolumeCommand.ts

```diff
@@ -0,0 +1,26 @@
+import CommandDefinition from '../domain/CommandDefinition';
+import IoConnection from '../clients/IoConnection';
+import VolumeEventMessage from "../domain/event-messages/VolumeEventMessage";
+
+async function volume(value: string) : Promise<string> {
+  const parsedInt: number = parseInt(value, 10);
+  const numericValue: number = isNaN(parsedInt) ? -1 : parsedInt;
+
+  if (isNaN(parsedInt) || (parsedInt < 0 || parsedInt > 100)) {
+    return `Nieprawidłowa głośność ${value}, podaj liczbę z zakresu 0-100`;
+  }
+
+  const eventMessage: VolumeEventMessage = { volume: numericValue };
```

Comment:
Use `parsedInt` instead.
```suggestion
  const eventMessage: VolumeEventMessage = { volume: parsedInt };
```